src/main/definitions/contact-applications.ts

```diff
@@ -15,5 +15,31 @@ export default [
 ];
 
 export const applicationTypes = {
+  a: [
+    'amend',
+    'Amend response',
+    'non-compliance',
+    'Claimant not complied',
+    'other',
+    'Contact the tribunal',
+    'order',
+    'Order other party',
+    'postpone',
+    'Postpone a hearing',
+    'publicity',
+    'Restrict publicity',
+    'strike',
+    'Strike out all or part of a claim',
+    'vary',
+    'Vary or revoke an order',
+  ],
+  b: [
+    'change-details',
+    'Change personal details',
+    'reconsider-decision',
+    'Consider a decision afresh',
+    'reconsider-judgement',
+    'Reconsider judgement',
+  ],
   c: 'witness',
```

Comment:
Why are duplicating values here (like 'amend' and 'Amend response')? And why isn't every value duplicated (like 'witness' for type c)?

Reply:
Some of the applications are respondent applications that come from ExUI, might need some cleaning to make more sense.

Reply:
Split data structure in to claimant and respondent